cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ProductVersionDTO.java

```diff
@@ -146,6 +146,9 @@
     @ApiModelProperty( value = "Cloud Providers enabled for the Product Version", accessMode = READ_ONLY )
     private List<UrlEntityDTO> cloudProviders;
 
+    @ApiModelProperty( "Whether a subscription from the product version can be killed by its customer" )
```

Comment:
```suggestion
    @ApiModelProperty( "Whether a subscription of this product version can be terminated before the natural expiration by its customer" )
```

not sure but looks better to me

Reply:
ok but we use "THE thing" not "this", so `of the product version`